app/pb_kits/playbook/pb_typeahead/typeahead.rb

```diff
@@ -5,12 +5,19 @@ module PbTypeahead
     class Typeahead
       include Playbook::Props
 
+      prop :async, type: Playbook::Props::Boolean,
+                    default: false
       prop :label
+      prop :load_options
```

Comment:
This is the *name or namespace* of the JavaScript function assigned to `window` which resolves a Promise, providing the XHR collected data from $anotherSource.